pages/news.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Image from "next/image";
+import { useFetchNews } from "./hooks/useFetchNews";
+import { poppins } from "../styles/fonts";
+
+interface NewsProp {
+  id: number;
+  title: string;
+  description: string;
+  url: string;
+  urlToImage: string;
+  name: string;
+}
+export default function News() {
+  const { data: news, isLoading, error } = useFetchNews();
+  console.log(news);
+  if (isLoading) {
+    return (
+      <div className="flex justify-center items-center">
+        <p>Loading News articles...</p>
+      </div>
+    );
+  }
+
+  if (error) {
+    return (
+      <div className="flex justify-center items-center">
+        <p>Error: {error?.message || "An unexpected error occurred."}</p>
+      </div>
+    );
+  }
+
+  return (
+    <div
+      className={`${poppins.className} flex justify-center items-center flex-col bg-[var(--main-bg-color-default)] text-[var(--main-color)] py-4`}
```

Comment:
Shouldn't you load the `poppins` className in your layout file?

Also, setting the `poppins` className like this does not change the font-family of the element.